glean-core/android/src/test/java/mozilla/telemetry/glean/private/LabeledMetricTypeTest.kt

```diff
@@ -316,19 +251,15 @@ class LabeledMetricTypeTest {
             subMetric = uuidMetric
         )
 
-        UuidsStorageEngine.record(labeledUuidMetric["label1"], UUID.randomUUID())
-        UuidsStorageEngine.record(labeledUuidMetric["label2"], UUID.randomUUID())
+        // labeledUuidMetric["label1"].record(UUID.randomUUID())
+        // labeledUuidMetric["label2"].record(UUID.randomUUID())
 
-        collectAndCheckPingSchema(Pings.metrics).getJSONObject("metrics")!!
-        */
+        // collectAndCheckPingSchema(Pings.metrics).getJSONObject("metrics")!!
     }
 
     @Ignore("String List metric not yet implemented")
```

Comment:
As above, unrelated to this bug, but either we remove labelled string lists from a-c, or we add them here.

Reply:
Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1570304